drf_extra_fields/fields.py

```diff
@@ -79,6 +79,13 @@ def get_file_extension(self, filename, decoded_file):
 
     def to_representation(self, file):
         if self.represent_in_base64:
+            # If the underlying ImageField is blank, a ValueError would be
+            # raised one `open`. When representing as base64, simply return an
+            # empty base64 str rather than let the exception propagate unhandled
+            # up into serializers.
+            if not file:
+                return ''
```

Comment:
Should we return `None` or `''` ?

Reply:
Apologies for the late review. Looks good to me too. Here we want to maintain the base64 compatibility and, [according to the specs](https://tools.ietf.org/html/rfc4648#section-10), `BASE64("") = ""` so it makes sense to return an empty string.

Reply:
I agree with @chpmrc , `''` allows for correct usage when using a serializer as well.